frontend/src/components/custom-tools/prompt-card/PromptOutput.jsx

```diff
@@ -374,6 +362,31 @@ function PromptOutput({
                       />
                     </div>
                   </div>
+                </Space>
+                <>
+                  <Divider className="prompt-card-divider" />
+                  <div className="prompt-card-result prompt-card-div">
+                    {isTableExtraction ? (
+                      <div />
```

Comment:
Is this empty div needed?

Reply:
@Deepak-Kesavan The empty div was previously necessary when we were rendering the table component separately. However, I've refactored the code for better quality and removed the need for the empty div. Thanks for the catch though!